protocols/kad/src/behaviour.rs

```diff
@@ -2674,6 +2683,12 @@ pub enum KademliaEvent {
     /// See [`Kademlia::kbucket`] for insight into the contents of
     /// the k-bucket of `peer`.
     PendingRoutablePeer { peer: PeerId, address: Multiaddr },
+
+    /// This peer's mode has been updated.
```

Comment:
```suggestion
    /// This peer's mode has been updated automatically.
```

To avoid some confusion where a user expects this event after a manual edit.